app/assets/stylesheets/modules/footer.scss

```diff
@@ -0,0 +1,44 @@
+@import "variables";
+
+.footer {
+
+  background: $blue;
+  height: 300px;
+  color: $white;
+
+  .header {
+    font-family: Lato;
+    padding-top: 20px;
+    text-align: center;
+  }
+
+  h2 {
+    font-family: Lato;
+    line-height: 1.4;
+    color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.